src/users/dto/users-query.dto.ts

```diff
@@ -32,8 +34,11 @@ export class UsersQueryDto extends PaginationArgsDto {
   @IsString()
   readonly search?: string;
 
-  @ApiPropertyOptional({ description: 'Event type filter', enum: EventType })
+  @ApiPropertyOptional({ description: 'Event types filter', enum: EventType })
   @IsOptional()
-  @IsEnum(EventType)
-  readonly event_type?: EventType;
+  @Transform(({ value }: TransformFnParams): unknown[] => {
+    return Array.isArray(value) ? (value as unknown[]) : [value];
```

Comment:
I'd kinda hope that this kinda assertion / manipulation wasn't needed? Surprising to me that the type system doesn't account for this directly.

Reply:
Do you mean casting to `unknown[]`? value is typed as `any` from 'class-validator' so you need to type it to match `unknown[]` as to not return any which will produce an eslint warning.

Reply:
I guess it's just weird to me how much the type system isn't adding anything here.